features/old/accounts/invoices.feature

```diff
@@ -39,22 +39,24 @@ Feature: Show invoices from account's page (#16015909)
 
   Scenario: Display years of invoices when there are invoices
     Given a provider is logged in on 1st January 2011
-    Given a buyer "zoidberg" signed up to provider "foo.3scale.localhost"
+    And a buyer "zoidberg" signed up to provider "foo.3scale.localhost"
     And the provider is charging its buyers in prepaid mode
     And an invoice of buyer "zoidberg" for January, 2011
-    And an invoice of buyer "zoidberg" for January, 2012
-    And I go to the invoices issued by me
-    Then I should see the list of years with invoices have the following years:
-      | 0  |
+    And an invoice of buyer "zoidberg" for January, 2014 on 1st January 2014 (without jobs)
```

Comment:
I actually added a `(without jobs)` option, because I noticed that otherwise all billing jobs between 2011 and 2014 get executed, and that takes too long.

And we don't need these jobs executed, we just do `on 1st January 2014` in order to avoid the billing period validation issue.